<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INSUPLTQAPT" failures="1" tests="3" name="ExtentReportTest" time="0.067" errors="0" timestamp="2024-08-11T19:23:34 IST">
  <testcase classname="extentreportgenaration.TestCase2" name="doLogin" time="0.011"/>
  <testcase classname="extentreportgenaration.TestCase2" name="doLogout" time="0.003">
    <skipped/>
  </testcase> <!-- doLogout -->
  <testcase classname="extentreportgenaration.TestCase2" name="doReg" time="0.006">
    <failure type="java.lang.AssertionError" message="Failing this Test">
      <![CDATA[java.lang.AssertionError: Failing this Test
at extentreportgenaration.TestCase2.doReg(TestCase2.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- doReg -->
</testsuite> <!-- ExtentReportTest -->
